flexmeasures/data/tests/test_dict_price_sensors.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime, timedelta
+import pytest
+import pytz
+from flexmeasures.data.models.planning.storage import (
+    StorageScheduler,
+)
+from flexmeasures.utils.calculations import integrate_time_series
+
+
+TOLERANCE = 0.00001
+
+
+@pytest.mark.parametrize(
+    "roundtrip_efficiency",
+    [
+        0.90,
+        0.95,
+    ],
+)
+def test_schedule(
```

Comment:
I would use a different name for this test, something like `test_schedule_multiple_price_sensors` and update the docstring to reflect that this test is using different price sensors.